src/cli/help.rs

```diff
@@ -11,7 +11,18 @@ pub(crate) fn rustup_help() -> String {
   for common platforms.
 
   If you are new to Rust consider running `rustup doc --book` to
-  learn Rust."
+  learn Rust.
+
+{HEADER}Common commands:{HEADER:#}
+
+    {LITERAL}$ rustup update{LITERAL:#}
+  
+  Update Rust toolchains and rustup
+
+    {LITERAL}$ rustup toolchain install{LITERAL:#}
```

Comment:
That's `$ rustup toolchain install stable` since you've mentioned the stable release.